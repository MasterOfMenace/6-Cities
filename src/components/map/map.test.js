import React from 'react';
import renderer from 'react-test-renderer';
import Map from './map.jsx';

const offersLocations = [
  [52.369553943508, 4.85309666406198],
  [52.369553943508, 4.85309666406198],
  [52.369553943508, 4.85309666406198],
  [52.369553943508, 4.85309666406198]
];

const cityLocation = [52.38333, 4.9];

const currentOfferLocation = [52.369553943508, 4.85309666406198];

it(`Првильное отображение компонента Map`, () => {
  const tree = renderer
    .create(<Map
      cityLocation={cityLocation}
      offersLocations={offersLocations}
      currentOfferLocation={currentOfferLocation}/>,
    {
      createNodeMock: () => document.createElement(`div`)
    })
    .toJSON();

  expect(tree).toMatchSnapshot();
});
