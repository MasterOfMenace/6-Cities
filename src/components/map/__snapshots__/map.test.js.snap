// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Првильное отображение компонента Map 1`] = `
<div
  id="map"
  style={
    Object {
      "height": "100%",
    }
  }
/>
`;
